free-2.md

```diff
@@ -58,12 +135,34 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Since these criteria are most easily described in natural language, it would
+be a significant challenge translating it into code.
+So a lot of time would need to be dedicated to syntax and naming.
+
+However, semantics would also be difficult as the program would need to
+interact with an email client in a secure way.
+Another challenge in semantics would be natural language processing problems 
+that may be required in order to filter things such as counting threads.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Due to the challenges in communicating with an email client and my unfamiliarity
+with NLP, this would be an ambitious project for me. The problem is quite open-
+ended so it would also be a challenge to determine how much the DSL should be
+able to handle.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a good idea because there would need to be a lot of language
+decisions in terms of syntax, since I would prefer to not use an XML
+configuration file interface.
+There would also be a lot of design decisions with naming.
+
+However, this project definitely tends towards the API side of the DSL spectrum.
+It is also a really ambitious project, in addition to all the work for 
+designing the language, there would have to be a lot of work to get the
+semantics to work.
```

Comment:
I agree, and I think that these factors maybe don't make for a good class project (but a super cool thing you could do on your own time!)